### ISSUE TITLE
Incorrect role is provided as button for "rich-text editor" formatting options such as bold, italic, and underline, rather than as "toggle button".: A11y_SQL Azure Data Studio_Notebook_AI4D

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26421]
### Tags:
#A11yTCS;#A11yWCAG;#WCAG4.1.2;#Name Role Value;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5. Tab till show/hide panel button and inspect it.
7. Observe that The name is not correctly defined for "show text label" button of the Dashboard. 
### Tags:
#A11yTCS;#A11yWCAG;#WCAG4.1.2;#Name Role Value;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1.Launch Azure Data Studio.
2. tab to File then New Notebook.
3. tab to Attach To and enter a SQL connection.
4. tab to + Text to add a text cell. Add some text.
5. tab to + Code to add a code cell. Add some code.
6. tab to formatting options and press enter to apply or remove the format on selected text.
7. Observe that Incorrect role is provided as button for "rich-text editor" formatting options such as bold, italic, and underline, rather than as "toggle button".

### Actual:
Incorrect role is provided as button for "rich-text editor" formatting options such as bold, italic, and underline, rather than as "toggle button".
### Expected:
The role should be as a toggle button for "rich-text editor" formatting options such as bold, italic, and underline, rather than as button. Therefore, NVDA should announce it as selected or pressed when the user selects or deselects a formatting option, and the format is applied to the selected text in the editor.

### User Impact:
Screen reader users cannot determine if any formatting has been applied to the selected text in the editor because the screen reader does not provide any state information about the options after they are activated.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 4.1.2: Name, Role, Value Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-4.1.2-Name-Role-Value-Guide.aspx)
- [MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

<img width="1365" height="708" alt="Image" src="https://github.com/user-attachments/assets/a0516eaa-c174-4258-aad6-e55a62a94f08" />

![Image](https://github.com/user-attachments/assets/2e63257f-e9bf-4a60-b9e1-5d4eade99f89)
